tools/sof-kernel-log-check.sh

```diff
@@ -229,6 +229,12 @@ ignore_str="$ignore_str"'|error: iteration [01]'
 ignore_str="$ignore_str"'|error: status'
 ignore_str="$ignore_str"'|error: cl_dsp_init: timeout HDA_DSP_SRAM_REG_ROM_STATUS read'
 
+# Buglink: https://github.com/thesofproject/sof-test/issues/564
+# found in CML and TGL
+#   for CML: e1000e 0000:00:1f.6 enp0s31f6: Hardware Error
+#   for TGL: e1000e 0000:00:1f.6 en0: Hardware Error
+ignore_str="$ignore_str"'|e1000e 0000:00:1f.6 en*: Hardware Error'
```

Comment:
maybe 0000:000:.+\..| en*: Hardware Error' ?

Reply:
Wow this works!